src/Testing/IntegrationTest.php

```diff
@@ -4,14 +4,14 @@
 
 namespace Tempest\Testing;
 
-use PHPUnit\Framework\TestCase as BaseTestCase;
+use PHPUnit\Framework\TestCase;
 use Tempest\AppConfig;
 use Tempest\Application\Kernel;
 use Tempest\Container\Container;
 use Tempest\Testing\Console\ConsoleCommandTester;
 use Tempest\Testing\Http\HttpRouterTester;
 
-abstract class TestCase extends BaseTestCase
+abstract class IntegrationTest extends TestCase
```

Comment:
I renamed this to `IntegrationTest`. I don't like it when people write `TestCase` and then have to find to right one based on the namespace.